train_llama.sh

```diff
@@ -0,0 +1,342 @@
+#!/bin/bash
+
+# set -x
+
+export GPU_MAX_HW_QUEUES=2
+export TORCH_NCCL_HIGH_PRIORITY=1
+
+
+# parsing input arguments
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+   export "$KEY"="$VALUE"
+done
+
+# Change for multinode config
+export CUDA_DEVICE_MAX_CONNECTIONS=1
+
+TEE_OUTPUT="${TEE_OUTPUT:-1}"
+NO_TORCH_COMPILE="${NO_TORCH_COMPILE:-0}"
+USE_FLASH_ATTN="${USE_FLASH_ATTN:-1}"
+NO_TRAINING="${NO_TRAINING:-0}" # NO_TRAINING=1: for computing metrics only
+ENABLE_PROFILING="${ENABLE_PROFILING:-0}"
+ENABLE_ROPE="${ENABLE_ROPE:-1}"
+ENABLE_ROPE_TE="${ENABLE_ROPE_TE:-1}"
+ENABLE_MOCK_DATA="${ENABLE_MOCK_DATA:-1}"
+DUMMY_RUN="${DUMMY_RUN:-0}"
+ADD_TASK="${ADD_TASK:-0}"
+LABEL="${LABEL:-"test"}"
+LOG_DIR="profile/${LABEL}"
+echo "NO_TRAINING=$NO_TRAINING"
+
+CWD=`pwd`
+GPUS_PER_NODE=`python -c "import torch; print(torch.cuda.device_count())"`
+
+
+# Change for multinode config
+MASTER_ADDR=localhost
```

Comment:
Do we intend to use this script for single-node only or do we also want to use it for multi-node? If latter, we'd better make those multi-node related options also be able to be specified from the command line.